packages/bpk-component-breadcrumb/package-lock.json

```diff
@@ -0,0 +1,278 @@
+{
+	"requires": true,
+	"lockfileVersion": 1,
+	"dependencies": {
+		"asap": {
+			"version": "2.0.6",
+			"resolved": "https://registry.npmjs.org/asap/-/asap-2.0.6.tgz",
+			"integrity": "sha1-5QNHYR1+aQlDIIu9r+vLwvuGbUY="
+		},
+		"bpk-component-button": {
```

Comment:
This package lock contains absolute version to backpack components as opposed to allowing lerna do it's thing.